pkg/receiver/rawk8seventsreceiver/receiver_test.go

```diff
@@ -408,6 +408,8 @@ func TestStorage(t *testing.T) {
 	host := storagetest.NewStorageHost(t, storageDir, "test")
 	assert.NoError(t, receiver.Start(ctx, host))
 
+	time.Sleep(10 * time.Millisecond)
```

Comment:
What's the reason behind this?

Reply:
This is to ensure that the second event actually comes in _after_ the first event.